src/sentry/api/endpoints/accept_organization_invite.py

```diff
@@ -110,6 +110,13 @@ class AcceptOrganizationInvite(Endpoint):
     def respond_invalid() -> Response:
         return Response(status=status.HTTP_400_BAD_REQUEST, data={"details": "Invalid invite code"})
 
+    @staticmethod
+    def respond_unauthorized() -> Response:
```

Comment:
does responding with 401 force re-login?

Reply:
from memory this makes some error message appear but does not force somebody to re-auth

Reply:
That's the goal—show a screen that lets the user know that they're signed into X account, then include a link for them to log out and be redirected back to the invite page.

Reply:
We include a link for the user to log out on the frontend.

Reply:
is this something we need to account for before we make this change?

Reply:
a video of the flow would help

Reply:
This is implemented in the frontend, so no changes for now. Currently, the frontend displays the same error view for all errors.